tests/testImageDecorrelation.py

```diff
@@ -245,19 +265,19 @@ def _testImages(self):
 
         # Expected (ideal) variance of difference image
         expected_var = self.svar + self.tvar
-        print('Expected variance:', expected_var)
+        print('EXPECTED VARIANCE:', expected_var)
```

Comment:
Since you have an assert about expected_var below, do you really need the print statement?